Automattic/wp-calypso
# migrate comments styles to webpack

#### Changes proposed in this Pull Request

* Migrate comments styles to webpack

#### Testing instructions

* Go to _My sites_ -> _Comments_
* Make sure nothing looks visually broken and exactly like on `master`

related #27515
